opendemic/map/model.py

```diff
@@ -0,0 +1,37 @@
+from config.config import CONFIG
+from opendemic.human.model import Human
+
+
+def get_risky_humans_geojson(
+	lat: float,
+	lng: float,
+	days_window: int = int(CONFIG.get('days_window')),
+	km_radius: int = int(CONFIG.get('km_radius'))
+) -> dict:
+	# get risky humans
+	risky_humans = Human.get_risky_humans(
+		lat=lat,
+		lng=lng,
+		days_window=days_window,
+		km_radius=km_radius
+	)
+
+	risky_humans_geojson = {
+		"type": "FeatureCollection",
+		"src": "https://raw.githubusercontent.com/beoutbreakprepared/nCoV2019/master/latest_data/latestdata.csv",
+		"features": []
+	}
+
+	for risky_human in risky_humans:
+		risky_humans_geojson["features"].append({
+			'type': 'Feature',
+			"properties": {
+				"mag": risky_human['mag']
+			},
+			'geometry': {
+				'type': 'Point',
+				'coordinates': [float(risky_human['longitude']), float(risky_human['latitude'])]
```

Comment:
It would be great to have a singe string for long and lat, since they are used in several places. It would be a good way to maintain one single variable, and avoid issues by using different naming conventions.

Reply:
Agreed and done.